crypto/fipsmodule/ec/make_tables.go

```diff
@@ -341,6 +412,35 @@ func bigIntToU64s(curve elliptic.Curve, n *big.Int) []uint64 {
 	return ret
 }
 
+// Convert big int to an array of 58-bit digits.
+// This is needed for P-521 Fiat-crypto implementation.
```

Comment:
```suggestion
// This is needed for P-521 Fiat-crypto implementation in fiat/p521_64.h.
```

Reply:
If you take the suggestion, I think it would be good to put `third_party` in the path, sorry.